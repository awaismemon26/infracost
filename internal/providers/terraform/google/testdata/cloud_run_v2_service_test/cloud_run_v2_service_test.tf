provider "google" {
  credentials = "{\"type\":\"service_account\"}"
  project     = "my-project"
  region      = "us-central1"
}

resource "google_cloud_run_v2_service" "throttling_enabled" {
  name     = "cloudrun-service-test-with-cpu-idle"
  location = "europe-west3"
  ingress = "INGRESS_TRAFFIC_ALL"

  template {
    containers {
      image = "us-docker.pkg.dev/cloudrun/container/hello"
    }
  }
}

resource "google_cloud_run_v2_service" "throttling_disabled" {
  name     = "cloudrun-service-test-with-cpu-idle"
  location = "europe-west3"
  ingress = "INGRESS_TRAFFIC_ALL"

  template {
    containers {
      image = "us-docker.pkg.dev/cloudrun/container/hello"
      resources {
        limits = {
          cpu = "1"
          memory = "512Mi"
        }
        cpu_idle = false
      }
    }
  }
}

