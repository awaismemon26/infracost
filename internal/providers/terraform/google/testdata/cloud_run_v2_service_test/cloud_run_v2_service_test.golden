
 Name  Monthly Qty  Unit  Monthly Cost 
                                       
 OVERALL TOTAL                   $0.00 
──────────────────────────────────
1 cloud resource was detected:
∙ 1 was estimated

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┓
┃ Project                                            ┃ Monthly cost ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━━━━━━━━━━┫
┃ TestCloudRunV2Service                              ┃ $0.00        ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━━━━━━━━━━┛